Mochiy Pop P One: Version 2.000; added; Mochiy Pop One: Version 2.000;

aaronbell opened the pull request:
9e975ad: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/edfac2b551595ac583bd1bb5519d18a8a86071cf.

 a1c78e4: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

 7ab92ae: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/edfac2b551595ac583bd1bb5519d18a8a86071cf.

 b0fb227: [gftools-packager] ofl/mochiypopone remove METADATA "source".  google/fonts#2587

davelab6 commented:
What's the difference between the two families?

davelab6 commented on ofl/mochiypopone/DESCRIPTION.en_us.html:
```diff
@@ -0,0 +1,3 @@
+<p>Mochiypop is a casual Gothic font created based on characters written by girls in their 20s in Japan.</p>
+<p>A cute display font, you can use it widely, such as for manga, magazines, movies, and signs.</p>
+<p>The included Kanji are modified versions of Source Han Sans.</p>
```
* The description should have a final paragraph like many do, linking to the upstream repo. 

* The description should link to sibling families. 

The cabin and Baloo 2 sets of families do those 2 things, as I recall. 

* We call it Noto Sans JP around these parts ;)

* I prefer tags on their own line, one sentence per line (no hand wraps), and a blank link as a final line; this makes diffs easier to read on GitHub and makes Unix piping better

aaronbell replied:
Done (I think)

davelab6 commented on ofl/mochiypopone/METADATA.pb:
```diff
@@ -0,0 +1,19 @@
+name: "Mochiy Pop One"
+designer: "FONTDASU"
+license: "OFL"
+category: "HANDWRITING"
```
It looks like informal sans to me, it isn't textured or calligraphic, so I recommend sans here

aaronbell replied:
Done

davelab6 commented on ofl/mochiypopone/METADATA.pb:
```diff
@@ -0,0 +1,19 @@
+designer: "FONTDASU"
+license: "OFL"
+category: "HANDWRITING"
+date_added: "2020-12-07"
```
They won't go live until 2020-12-14 so better to use that date

aaronbell replied:
Done

davelab6 commented on ofl/mochiypopone/METADATA.pb:
```diff
@@ -0,0 +1,19 @@
+  full_name: "Mochiy Pop One Regular"
+  copyright: "Copyright 2020 The MochiyPop Project Authors (https://github.com/fontdasu/Mochiypop)"
+}
+subsets: "chinese-simplified"
```
There's no chinese so remove that

aaronbell replied:
From what I can tell, there's no way to create the METADATA.pb file locally and upload to the server via the gftools packager. Is the only way to edit this after I run packager?

davelab6 commented on ofl/mochiypopone/METADATA.pb:
```diff
@@ -0,0 +1,19 @@
+  copyright: "Copyright 2020 The MochiyPop Project Authors (https://github.com/fontdasu/Mochiypop)"
+}
+subsets: "chinese-simplified"
+subsets: "chinese-traditional"
```
Remove

aaronbell replied:
Done

aaronbell commented:
The key difference is that in the `P` variant, the proportional versions of the kana are used by default, whereas in the non-`P` variant, the full-width versions are used by default. 

This was a special request by the original designer because he envisions this font to be used in applications such as Youtube Movie and Adobe Premiere Pro which cannot support OpenType fully.

davelab6 commented:
Please update the description to explain it when interlinking

davelab6 commented:
How strong is the "P" family naming as a convention for this (significant) difference, in the overall japanese fonts marketplace?

aaronbell commented:
@davelab6 Seems to be pretty common
```
“P” stands for Proportion
You’ve probably seen the letter “P” next to a font while you were searching for one that matches your requirements. In this case, you’ve stumbled upon a font where each character has been condensed independently, which results in tighter letter-spacing.
```

https://www.1stopasia.com/blog/ever-wondered-which-japanese-fonts-you-should-use/

m4rc1e commented:
Hey Aaron,

Thank you for using the packager and reading our spec.



> The key difference is that in the `P` variant, the proportional versions of the kana are used by default, whereas in the non-`P` variant, the full-width versions are used by default.
> 
> This was a special request by the original designer because he envisions this font to be used in applications such as Youtube Movie and Adobe Premiere Pro which cannot support OpenType fully.

Is this "P" variant going to be implemented in other families?

m4rc1e reviewed (changes requested):
There are overlap issues.

<img width="428" alt="Screenshot 2020-12-08 at 11 19 19" src="https://user-images.githubusercontent.com/7525512/101477532-40597d80-3947-11eb-9969-e0bf2332581a.png">

_OSX 10.15.7 Chrome v86 (very new)_

Is there a reason why the overlaps are not removed?

aaronbell commented:
@m4rc1e 
> Is this "P" variant going to be implemented in other families?
I believe this is the only example of such in this set of fonts, but it could certainly be possible in future families. One thing I should note, however, is that the Shippori Antique and Shippori Mincho families include a "B1" variant which has a 'round' filter applied. I've seen the term "B1" mentioned with regards to other fonts, though we could potentially change it to "rounded" if you prefer. 

> Is there a reason why the overlaps are not removed?
No, they should have been removed. I'll check my build script.

aaronbell commented:
Updated

## Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added; Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

---
 53db6dd: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/edfac2b551595ac583bd1bb5519d18a8a86071cf.

 c46e0c0: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

 c21bb5d: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/edfac2b551595ac583bd1bb5519d18a8a86071cf.

 842e074: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

 5953669: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/edfac2b551595ac583bd1bb5519d18a8a86071cf.

 bbc36c2: [gftools-packager] ofl/mochiypopone remove METADATA "source".  google/fonts#2587

aaronbell commented:
Updated

## Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added; Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

---
 53db6dd: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/edfac2b551595ac583bd1bb5519d18a8a86071cf.

 c46e0c0: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

 c21bb5d: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/edfac2b551595ac583bd1bb5519d18a8a86071cf.

 842e074: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

 5953669: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/edfac2b551595ac583bd1bb5519d18a8a86071cf.

 bbc36c2: [gftools-packager] ofl/mochiypopone remove METADATA "source".  google/fonts#2587

 4301be0: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/e21936b8ae3883fe7b3a2b973b2ebd1f77c7821a.

 46a5fd1: [gftools-packager] ofl/mochiypopone remove METADATA "source".  google/fonts#2587

 53a562f: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/e21936b8ae3883fe7b3a2b973b2ebd1f77c7821a.

 fc908ae: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

aaronbell commented:
Sorry for the constant pinging. Figuring some things out :)

m4rc1e commented:
Overlaps are still present in some glyphs.

<img width="275" alt="Screenshot 2020-12-09 at 11 45 30" src="https://user-images.githubusercontent.com/7525512/101625916-0d30f000-3a14-11eb-873c-a4e09f68d90e.png">

aaronbell commented:
:/ you’d think “removeOverlaps=True” would take care of these things. I’ll figure out what’s happening.

aaronbell commented:
@m4rc1e Can you confirm you're looking at the most recent version of the TTFs?

<img width="912" alt="Screen Shot 2020-12-09 at 8 33 59 AM" src="https://user-images.githubusercontent.com/8460297/101657950-558ee480-39f9-11eb-8933-8dace00932b8.png">

Opening the repository versions in OTMA, I don't see any overlaps.

aaronbell commented:
Updated

## Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

---
 68a281d: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/78b9cd5d0772e74c3369db7425f12cf62d5bceff.

 6a4cedb: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

aaronbell commented:
Updated

## Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added; Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

---
 f34dc5a: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/78b9cd5d0772e74c3369db7425f12cf62d5bceff.

 23a78c7: [gftools-packager] ofl/mochiypopone remove METADATA "source".  google/fonts#2587

 8905222: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/78b9cd5d0772e74c3369db7425f12cf62d5bceff.

 300a3be: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

aaronbell commented:
Designer found a bug in the P version. Updated.

aaronbell commented:
Please hold on reviewing this font family. There's some further issues to sort out :/

aaronbell commented:
Updated

## Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added; Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

---
 09e668a: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/a5873d5e4e5bbe35d5aac839b15df991081534e2.

 bf9d774: [gftools-packager] ofl/mochiypopone remove METADATA "source".  google/fonts#2587

 5e7127a: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/a5873d5e4e5bbe35d5aac839b15df991081534e2.

 b574466: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

aaronbell commented:
Updated

## Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added; Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

---
 0c1611c: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/caa4ced1c7b1a364cce1771b43869248523b0be0.

 a40604d: [gftools-packager] ofl/mochiypopone remove METADATA "source".  google/fonts#2587

 171f3a8: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/caa4ced1c7b1a364cce1771b43869248523b0be0.

 071ef97: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

aaronbell commented:
Updated

## Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added; Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

---
 6f5e7ff: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/7a461e24d6c6954034af27cd14ccbd6d70a4d59d.

 aed9aab: [gftools-packager] ofl/mochiypopone remove METADATA "source".  google/fonts#2587

 bc0bce0: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/7a461e24d6c6954034af27cd14ccbd6d70a4d59d.

 664b5db: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

aaronbell commented:
The designer came back with some additional changes. Sorry again for the various updates. He's happy, I'm happy. :)

aaronbell commented:
Updated

## Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added; Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

---
 8e44f7a: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/65b4795227ef13c45acb07c5e0d1c3fd61c0b574.

 39434b1: [gftools-packager] ofl/mochiypopone remove METADATA "source".  google/fonts#2587

 135ce11: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/65b4795227ef13c45acb07c5e0d1c3fd61c0b574.

 effbb97: [gftools-packager] ofl/mochiypoppone remove METADATA "source".  google/fonts#2587

aaronbell commented:
Updated

## Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added; Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

---
 1567f7a: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/2c240f5bc8a60bc73979beaed13684d911abe93c.

 207b746: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/2c240f5bc8a60bc73979beaed13684d911abe93c.

aaronbell commented:
Updated

## Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added; Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

---
 066b51c: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/896061c9469171837d0dafaaffe07d5eb857dc2d.

 b7eccdb: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/896061c9469171837d0dafaaffe07d5eb857dc2d.

aaronbell commented:
Updated

## Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added; Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

---
 21c2b90: [gftools-packager] Mochiy Pop One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/9641b84721cc06c4765b4ba2d8de690d50c411db.

 a1b0931: [gftools-packager] Mochiy Pop P One: Version 2.000; ttfautohint (v1.8.3) added

* Mochiy Pop P One Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/Mochiypop.git at commit https://github.com/fontdasu/Mochiypop/commit/9641b84721cc06c4765b4ba2d8de690d50c411db.

aaronbell commented:
Pushed an update per Foundry request.

aaronbell commented:
Fonts are dehinted. Space savings :)

davelab6 commented:
Thanks @aaronbell :) Please rebase to allow merging

aaronbell commented:
@davelab6 Done!

RosaWagner commented:
Was about to merge and just saw this in Mochi Pop One 

![Capture d’écran 2021-07-29 à 11 52 48](https://user-images.githubusercontent.com/12222436/127472088-ebc9400f-879b-4651-bde4-60ceea932519.png)

aaronbell commented:
Wow! Good catch. I saw that the Eth also had the same issue and corrected it in both. Skimmed the rest of the Latin / symbol set and didn't see anything amiss. 

New version uploaded.

RosaWagner commented:
LGTM
[mochiypoppone_qa.zip](https://github.com/google/fonts/files/6938134/mochiypoppone_qa.zip)
